lib/src/index.ts

```diff
@@ -282,7 +282,7 @@ export { default as NxThreatIndicatorLegend, NxThreatIndicatorLegendProps }
 export {
   default as NxSearchDropdown,
   Props as NxSearchDropdownProps,
-  SEARCH_DEBOUNCE_TIME as NX_SEARCH_DROPDOWN_DEBOUNCE_TIME
+  SEARCH_DEBOUNCE_TIME as NX_STANDARD_DEBOUNCE_TIME
```

Comment:
We'll still need the old export to so as to not break backwards compatibility

Reply:
Thanks for the reminder. Updated here 409fb10d5bde24b679bacaa0dd79d0a09a2e4dd2.